source/release-notes/changelogs/backend/changelog-backend-2022.rst

```diff
@@ -1,7 +1,20 @@
+.. _backend_20220607:
+
+7 June 2022 Release
+~~~~~~~~~~~~~~~~~~~
+
+- Released Flexible Sync as GA.
+- Released Data API as GA.
+- Introduced field-level permissions for Flexible Sync.
+- Released Asymmetric Sync, which optimizes Flexible Sync for write-heavy workloads.
+- Introduced Data API for any App.
+- Introduced local regions for Azure data sources.
+- Introduced Atlas Alerts for failed requests that have met a certain threshold.
```

Comment:
i find the wording here kinda awkward and opaque. 

- what type of requests? 
- is the threshold user-defined? 

could this be rephrased?

Reply:
```suggestion
```